lib/ArduinoNative/SocketServer.cpp

```diff
@@ -1,444 +0,0 @@
-/* MIT License

```

Comment:
Not sure why, but I cannot see that SocketServer.h was deleted. It is not longer in the file tree, but it does not appear in the review diff